install/README.md

```diff
@@ -484,22 +484,79 @@ These variables can be set to allow AWS authentication for S3 storage spaces as
-`/var/log/archivematica/storage-service/storage_service.log`. Starting with
-Storage Service 0.11.0, logging configuration defaults to using stdout and
-stderr for all logs. If no changes are made to the new default configuration
-logs will be handled by whichever process is managing Archivematica's services.
-For example on Ubuntu 16.04, Ubuntu 18.04 or CentOS 7, Archivematica's processes
-are managed by systemd. Logs for the Storage Service can be accessed using
-`sudo journalctl -u archivematica-storage-service`. When running Archivematica
-using docker, `docker-compose logs` commands can be used to access the logs from
-different containers.
-
-The Storage Service will look in `/etc/archivematica` for a file called
-`storageService.logging.json`, and if found, this file will override the default
-behaviour described above.
-
-The [`storageService.logging.json`](./storageService.logging.json) file in this
-directory provides an example that implements the logging behaviour used in
-Storage Service 0.10.0 and earlier.
+Logging configuration defaults for all logs to using `stdout` and `stderr`
+unless they are configured to do otherwise. If there are no changes to the
+default configuration they will be handled by whichever process is managing
+Archivematica's services. For example, on Ubuntu 16.04, Ubuntu 18.04 or
+CentOS 7, Archivematica's processes are managed by `systemd`. Logs for the
+Storage Service can be accessed using `sudo journalctl -u archivematica-storage
+-service`.
+
+When running Archivematica using docker, `docker-compose logs` commands can be
+used to access the logs from different containers, e.g. `docker-compose logs
+-f archivematica-storage-service`.
+
+### Overriding the logging configuration
+
+Via the Django configuration settings, i.e. [base.py][django-config], the
+storage service will look for a file in `/etc/archivematica/` called
+`storageService.logging.json`. If this file is found it can be used to override
+the default logging behavior.
+
+The [`storageService.logging.json`](./storageService.logging.json) file found
+in this installation directory provides an example that is configured to output
+to a logs directory: `/var/log/archivematica/storage-service`, i.e.
+`/var/log/archivematica/storage-service/storage_service.log`.
+
+### Increase or decrease the logging output
+
+Archivematica uses Python's standard approach to logging. There is a hierarchy
+of logging levels, at each level, more or less output can be configured. The
+values run from DEBUG (verbose) to CRITICAL (less verbose) as follows:
+
+* DEBUG.
+* INFO.
+* WARNING.
+* ERROR.
+* CRITICAL.
+
+The [Python documentation][python-docs] provides greater explanation.
+
+Though best efforts are taken to include the most useful information for
+debugging as possible. Your mileage may vary in debugging Archivematica or the
+Storage Service depending on the way the developer has written any particular
+module.
```

Comment:
I'd make this one sentence - the first sentence reads more like a fragment than a full sentence at present.